apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -156,8 +156,9 @@ export class BulkScriptAction extends BaseScriptAction {
         parameterLongName: '--parallelism',
         parameterShortName: '-p',
         argumentName: 'COUNT',
-        description: 'Specify the number of concurrent build processes'
-          + ' The value "max" can be specified to indicate the number of CPU cores.'
+        environmentVariable: 'RUSH_PARALLELISM',
```

Comment:
Should this just reference the enum from `EnvironmentConfiguration`?

Reply:
Fixed